src/lib.rs

```diff
@@ -1049,6 +1049,17 @@ impl Connection {
         Ok(done)
     }
 
+    /// Return idle timeout value
+    fn get_idle_timeout(&mut self) -> time::Duration {
+        let idle_timeout = cmp::max(
+            time::Duration::from_millis(self.local_transport_params.idle_timeout),
+            3 * self.recovery.pto(),
+        );
+        trace!("{} idle timeout = {:?}", self.trace_id, idle_timeout);
```

Comment:
Do we really need this log? it seems like it would be quite spammy since it would log for almost every received and sent packets.

Reply:
It was for debugging (used trace!()) but I can remove it.